gdal/ogr/ogrsf_frmts/cad/GNUmakefile

```diff
@@ -0,0 +1,20 @@
+
+include ../../../GDALmake.opt
+
+SUBDIRS-yes := libopencad
+
+OBJ	= \
+	ogrcaddriver.o \
+	gdalcaddataset.o \
+	ogrcadlayer.o \
+	vsilfileio.o
+
+CPPFLAGS	:=	-I. -I.. -I../.. $(CPPFLAGS) -std=c++11 
```

Comment:
You need to AC_SUBST() the variable in configure.in and add a place holder in GDALmake.opt.in